.github/workflows/preliminary_checks.yml

```diff
@@ -70,3 +75,23 @@ jobs:
             echo "\`\`\`" >> $GITHUB_STEP_SUMMARY
             exit 1
           fi
+
+  title-check:
+    name: CC PR Title
```

Comment:
let's change this to "PR Title Format" to make it a bit more intuitive. What's is the "CC" prefix anyway?

Reply:
ah, "conventional commits"! But made my point that +90% won't know what that means :)